src/util/action-hooks.js

```diff
@@ -0,0 +1,24 @@
+exports.ACTION_HOOKS = [
+  {
+    trustDomain: 'gecko',
+    level: '1',
+    actionPerm: 'generic',
+    groups: ['active_scm_level_1'],
+    // inputSchema,
+  },
+  {
+    trustDomain: 'comm',
+    level: '1',
+    actionPerm: 'generic',
+    groups: ['active_scm_level_1'],
+    // inputSchema,
+  },
+  {
+    trustDomain: 'gecko',
+    level: '1',
+    actionPerm: 'purge-caches',
+    groups: ['taskcluster', 'vpn_sheriff'],
```

Comment:
I see no reason this should be so limited :)

But it's a good example nonetheless...

Reply:
Yeah, we can work out the permissions later -- this is almost just placeholder stuff right now.